reference/forms/types/choice.rst

```diff
@@ -95,23 +95,31 @@ method::
             new Category('Cat3'),
             new Category('Cat4'),
         ],
-        'choice_label' => function(Category $category, $key, $value) {
-            return strtoupper($category->getName());
+        // a property path can by used to define the options below
```

Comment:
I'm afraid I don't understand this comment:

```
a property path can by used to define the options below
```

The value of the `choice_value` can be a property path instead of just a simple property name? If that's true ... why do we add "... to define the options below" ? I don't understand what do we try to say here.

Reply:
`property path` !== `property name`.
A property path like `state` may be a property or `getState() | hasState() | isState()`.
I'm trying to say that all the examples below may use it instead of a closure.
WDYT of:
```php
// as done for the "choice_value", a property path can by used instead of
// a closure  for all the "choice_*" options below

Reply:
Yes! I understand this comment now. Thanks.

Reply:
Thank you.